### DISCUSSION TITLE
Possible to keep repeated words

### DISCUSSION THREAD

**Discussion by section33** [openai/whisper#304] [Q&A] [answered]
Trying to generate a super accurate transcription of exactly what's being said and noticed that if the speaker repeats a word it only shows up as one instance. Is it possible to bypass whatever filtering is happening there?

**Comment by jianfch** [comment, marked as answer]
The decoding heuristics does not appear to suppress repeated words. So the "filtering" is performed by the model. Typical transcripts done by human omit repeated words. As a result, it has likely learned this from its training data. One way to "bypass this filtering" is to fine-tune the model on data that does not omit repeating words.
